CI.md

```diff
@@ -125,6 +125,13 @@ Service Account shall have following permissions ([IAM roles](https://cloud.goog
 | Java Wordcount Direct Runner | Runs Java WordCount example with Direct Runner.                                               | Yes              | Yes                   | Yes           | -                        |
 | Java Wordcount Dataflow      | Runs Java WordCount example with DataFlow Runner.                                             | -                | Yes                   | Yes           | Yes                      |
 
+### PostCommit Workflows
+
+#### Python Sickbay - [job-postcommit-python-sickbay.yml](.github/workflows/job-postcommit-python-sickbay.yml)
```

Comment:
As mentioned in other PRs, please make this part of a single table

Reply:
Done!